pkg/graph/schema.resolvers.go

```diff
@@ -248,7 +400,24 @@ func (r *Resolver) Mutation() generated.MutationResolver { return &mutationResol
 // Query returns generated.QueryResolver implementation.
 func (r *Resolver) Query() generated.QueryResolver { return &queryResolver{r} }
 
+// SystemIntake returns generated.SystemIntakeResolver implementation.
+func (r *Resolver) SystemIntake() generated.SystemIntakeResolver { return &systemIntakeResolver{r} }
+
 type accessibilityRequestResolver struct{ *Resolver }
 type accessibilityRequestDocumentResolver struct{ *Resolver }
 type mutationResolver struct{ *Resolver }
 type queryResolver struct{ *Resolver }
+type systemIntakeResolver struct{ *Resolver }
+
+// !!! WARNING !!!
```

Comment:
Anything below here can probably be deleted. It tends to accumulate when iterating on the schema since `gqlgen` tries not to remove code it doesn't recognize.